docs/framework/performance/lazy-initialization.md

```diff
@@ -78,35 +78,35 @@ manager: "wpickett"
+ Exception caching is enabled when you use any <xref:System.Lazy%601?displayProperty=fullName> constructor that takes an initialization method (`valueFactory` parameter); for example, it is enabled when you use the `Lazy(T)(Func(T))`constructor. If the constructor also takes a <xref:System.Threading.LazyThreadSafetyMode> value (`mode` parameter), specify <xref:System.Threading.LazyThreadSafetyMode.ExecutionAndPublication?displayProperty=fullName> or <xref:System.Threading.LazyThreadSafetyMode.PublicationOnly?displayProperty=fullName>. Specifying an initialization method enables exception caching for these two modes. The initialization method can be very simple. For example, it might call the default constructor for `T`: `new Lazy<Contents>(() => new Contents(), mode)` in C#, or `New Lazy(Of Contents)(Function() New Contents())` in Visual Basic. If you use a <xref:System.Lazy%601?displayProperty=fullName> constructor that does not specify an initialization method, exceptions that are thrown by the default constructor for `T` are not cached. For more information, see the <xref:System.Threading.LazyThreadSafetyMode> enumeration.  
   
 > [!NOTE]
->  If you create a <xref:System.Lazy%601> object with the `isThreadSafe` constructor parameter set to `false` or the `mode` constructor parameter set to <xref:System.Threading.LazyThreadSafetyMode?displayProperty=fullName>, you must access the <xref:System.Lazy%601> object from a single thread or provide your own synchronization. This applies to all aspects of the object, including exception caching.  
+>  If you create a <xref:System.Lazy%601> object with the `isThreadSafe` constructor parameter set to `false` or the `mode` constructor parameter set to <xref:System.Threading.LazyThreadSafetyMode.None?displayProperty=fullName>, you must access the <xref:System.Lazy%601> object from a single thread or provide your own synchronization. This applies to all aspects of the object, including exception caching.  
   
- As noted in the previous section, <xref:System.Lazy%601> objects created by specifying <xref:System.Threading.LazyThreadSafetyMode?displayProperty=fullName> treat exceptions differently. With <xref:System.Threading.LazyThreadSafetyMode>, multiple threads can compete to initialize the <xref:System.Lazy%601> instance. In this case, exceptions are not cached, and attempts to access the <xref:System.Lazy%601.Value%2A> property can continue until initialization is successful.  
+ As noted in the previous section, <xref:System.Lazy%601> objects created by specifying <xref:System.Threading.LazyThreadSafetyMode?displayProperty=fullName> treat exceptions differently. With <xref:System.Threading.LazyThreadSafetyMode.PublicationOnly>, multiple threads can compete to initialize the <xref:System.Lazy%601> instance. In this case, exceptions are not cached, and attempts to access the <xref:System.Lazy%601.Value%2A> property can continue until initialization is successful.  
```

Comment:
<xref:System.Threading.LazyThreadSafetyMode?displayProperty=fullName> --> <xref:System.Threading.LazyThreadSafetyMode.PublicationOnly?displayProperty=fullName> (The first use of LazyThreadSafetyMode)